test/helpers/mocks.lua

```diff
@@ -0,0 +1,55 @@
+local M = {}
+
+local validate = function (mocks)
```

Comment:
It is just a codestyle used by us. The same for all methods here.
```suggestion
local validate = function(mocks)
```

Reply:
Formatted.